Q1

(1) the idea can be examined and tested statistically is that this idea is measurable and quantitive. For example, just like the first vedio, the price of apples can be examined, but which government does better is not quantitive.
(2)The key "criteria" defining what a good null hypothesis is that the hypothesis is measurable and quantitive, which means that can be examined and tested statistically.
(3) the null hypothesis we choose is always equal to a certain number or certain circumstance, and for the alternative hypothesis is the opposite circumstance, greater, less or not equal to the certain number for null hypothesis. We use alternative hypothesis to reject the null hypothesis.

Q2

xi is the individual data in the dataset, x-is the mean of the sample datasets. 𝜇 is the population mean, and 𝜇0 is the hypothesis population mean.
we use the xi to calculate x-, and use the sampling to analysis the probability that the 𝜇0 is ridiculus.we use the sample statistic (like 
x- as an estimate to make inferences about the population parameter (like μ)The main reason for these steps is to analysis our 𝜇0 hypothesis is statisitically false or true related to the population mean.

Q3

as the sampling distribution helps us connect sample data (which comes from the real world) to broader population-level inferences, the relavence of sampling disribution gives us the likelihood of our test statistic under the null hypothesis. For our hypothesis test, the goal for the p-value is to use to reject or not reject the null hypothesis. p-value is the probability for our hypothesis to be extreme. So, consider the real world to think about whether our hypothesis is ridiculous is the thing that our hypothesis test do the same.

Q4

The p-value measures the probability of the extreme or more extreme values get assuming that the null hypothesis is true.If the observed test statistic looks extreme to the sampling distribution of the test statistic under the null hypothesis, we say that p-value is large.

Q5

H0:p = 0.5 p is the possibility for a man to tilt head.
Ha:p != 0.5 

In [1]:
import numpy as np

# Parameters
num_couples = 124  # Total number of couples
right_tilt_observed = 80  # Observed number of right tilts
simulations = 10000  # Number of simulations to run

# Simulate flipping a coin (0 = left tilt, 1 = right tilt) for 124 couples over 10,000 trials
simulated_tilts = np.random.binomial(num_couples, 0.5, simulations)

# Count how many simulations resulted in 80 or more right tilts
extreme_cases = np.sum(simulated_tilts >= right_tilt_observed)

# Calculate the p-value as the proportion of extreme cases
p_value = extreme_cases / simulations

# Output the result
print(f"Simulated p-value: {p_value}")

Simulated p-value: 0.0012


0.01≥𝑝>0.001
 
Strong evidence against the null hypothesis

Here’s a summary of our chat history:

1. **Statistical Testing Concepts**:
   - We discussed the importance of distinguishing between ideas that can be statistically examined and tested versus those that cannot.
   - You noted that outcomes of tests refer to the population parameter rather than the sample statistic, highlighting the relevance of the sampling distribution of the test statistic under the null hypothesis.

2. **P-Value Understanding**:
   - We explored the concept of p-values, including their definition and what they represent in hypothesis testing.
   - You expressed confusion about the meaning of extreme or more extreme values in the context of p-values.
   - I clarified that a smaller p-value indicates more extreme data relative to the null hypothesis, leading to the rejection of \( H_0 \).

3. **Kissing Couples Study**:
   - You referenced a study by Güntürkün (2003) on head tilting in kissing couples, where 80 out of 124 couples tilted their heads to the right (64.5%).
   - We discussed simulating a p-value using a "50/50 coin-flipping" model under the null hypothesis that there is no head-tilt tendency.
   - I performed the simulation in Python, which yielded a p-value of approximately **0.001**, suggesting strong evidence against the null hypothesis.

4. **Python Simulation Code**:
   - I provided you with the full Python code used for the simulation, explaining each part of the code and its function in estimating the p-value.

If you have any more questions or if there’s anything else you’d like to discuss, feel free to ask!

https://chatgpt.com/share/6705fe10-5588-8006-a15f-fd18d3b73efa

Q6

(1) A smaller p-value cannot definitively prove that the null hypothesis is false. p-value is small just means that we have strong statistically evidence suggests that we reject null hypothesis, but the there are too many uncertainty for the real circumstances.
(2)No, because we only said that like we are 95% confidence that fido is innocent by applying p-value.
(3)Similarly, we cannot definitely prove that fido is guilty. Even though the p-value is greater, we just say we failed to reject our null hypothesis, cannot say support the other side.
(4)Whatever the p-value is low or high, we cannot promise our hypothesis is definitely true in the real circumstance. We only say that we have statistical evidence to prove, but about the real world there are too many uncertainty.

Q7

the orginal code:

In [3]:
import pandas as pd

patient_data = pd.DataFrame({
    "PatientID": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
    "Age": [45, 34, 29, 52, 37, 41, 33, 48, 26, 39],
    "Gender": ["M", "F", "M", "F", "M", "F", "M", "F", "M", "F"],
    "InitialHealthScore": [84, 78, 83, 81, 81, 80, 79, 85, 76, 83],
    "FinalHealthScore": [86, 86, 80, 86, 84, 86, 86, 82, 83, 84]
})

import numpy as np

np.random.seed(1)  # make simulation reproducible
number_of_simulations = 10000  # experiment with this... what does this do?
n_size = len(patient_data)  # 10
IncreaseProportionSimulations_underH0random = np.zeros(number_of_simulations)

# generate "random improvement" proportions assuming H0 (vaccine has no average effect) is true 
# meaning that the "before and after" differences are positive or negative at "random"
for i in range(number_of_simulations):
    
    # why is this equivalent to the suggested idea above?
    random_improvement = np.random.choice([0,1], size=len(patient_data), replace=True)  # <<< `replace=True` ^^^

    # why is .mean() a proportion? 
    IncreaseProportionSimulations_underH0random[i] = random_improvement.mean()
    
# Step 1: Calculate the observed improvement proportion
patient_data['HealthScoreChange'] = patient_data['FinalHealthScore'] - patient_data['InitialHealthScore']
observed_improvement_proportion = (patient_data['HealthScoreChange'] > 0).mean()

# Step 2: Count how many of the simulated proportions are greater than or equal to the observed proportion
extreme_count = (IncreaseProportionSimulations_underH0random >= observed_improvement_proportion).sum()

# Step 3: Calculate the p-value
p_value = extreme_count / number_of_simulations

print("Observed Improvement Proportion:", observed_improvement_proportion)
print("P-value:", p_value)


Observed Improvement Proportion: 0.8
P-value: 0.0565


the changed code:

In [8]:
# Step 1: Calculate the observed improvement proportion
patient_data['HealthScoreChange'] = patient_data['FinalHealthScore'] - patient_data['InitialHealthScore']
observed_improvement_proportion = (patient_data['HealthScoreChange'] > 0).mean()

# Step 2: Count how many of the simulated proportions are greater than the observed proportion
one_sided_extreme_count = (IncreaseProportionSimulations_underH0random > observed_improvement_proportion).sum()

# Step 3: Calculate the one-sided p-value
one_sided_p_value = one_sided_extreme_count / number_of_simulations

print("Observed Improvement Proportion:", observed_improvement_proportion)
print("One-Sided P-value:", one_sided_p_value)


Observed Improvement Proportion: 0.8
One-Sided P-value: 0.0107


we changed extreme_count = (IncreaseProportionSimulations_underH0random >= observed_improvement_proportion).sum()
into one_sided_extreme_count = (IncreaseProportionSimulations_underH0random > observed_improvement_proportion).sum()
that means we cannot use the proportions which are equal to the observed proportion

the interpretation of our alternitave hypothesis for two sides is that the probability of the result could be an improvement or a decline, but the one side is that the result can only an improvement.

we should indeed expect the p-value to be smaller in the "one tailed" versus "two tailed" analysis.That's because the probability for equal (have no effect) is include in two tailed, exclude in one tailed, so the probability for an extreme value for equal or greater should be greater than an extreme value for just greater.

Q8

H0: p=0.5 make a hypothesis that students are guess randomly,p is the proportion of student guess correctly
Ha： p n> 0.5  means that students are not guess randomly, they have the ability


In [9]:
import numpy as np

n_students = 80 
p_guess = 0.5    
observed_correct = 49  
n_simulations = 100000  

np.random.seed(0)  

simulated_correct = np.random.binomial(n_students, p_guess, n_simulations)

# Calculate the proportion of simulations where the number of correct guesses >= 49
p_value_simulation = np.mean(simulated_correct >= observed_correct)
print(f"Simulated p-value: {p_value_simulation}")

Simulated p-value: 0.02849


Assuming H0 is true, there is a 0.02849 probablility of getting a greater than 49/80 by chance
since 0.02849<0.5, that means there are statistical evidence to reject that students dont have ability to distinguish the drinks.


Q9

yes

Here's a summary of our chat:

1. **Code Interpretation**: You shared some code related to analyzing patient health score changes. I explained how to interpret the code, focusing on the comparison of actual health improvements against random changes simulated under the null hypothesis (H0), which assumes no effect of treatment (e.g., vaccine).

2. **Calculating p-value**: You asked how to calculate the p-value. I guided you through computing the p-value by:
   - Simulating random data under the null hypothesis.
   - Comparing the observed improvement proportion with the distribution of simulated improvements.
   - Computing the p-value as the proportion of simulations that show a more extreme result than the observed data.

3. **One-Sided vs. Two-Sided Tests**:
   - We discussed the difference between one-sided (one-tailed) and two-sided (two-tailed) hypothesis tests.
   - **One-sided test**: Focuses on whether there is a significant improvement (e.g., health improvement due to a treatment).
   - **Two-sided test**: Considers whether the data is significantly different from the null hypothesis in either direction (improvement or decline).

4. **Coding for One-Sided and Two

https://chatgpt.com/share/6706ba2d-bd68-8006-8913-2f62c124cec0